test/07-blink.spec.ts

```diff
@@ -13,7 +13,6 @@ describe('blink', () => {
 	});
 
 	it('writes to a file that represents the LED, and writes a 0 at the end to turn the LED off', async () => {
-		// TODO: Fix the typings for blink
```

Comment:
`blinking` was updated to v0.0.4 with TypeScript support recently, which resolves this TODO.